xbmc/application/ApplicationVolumeHandling.h

```diff
@@ -27,6 +37,8 @@ class CApplicationVolumeHandling
   void SetMute(bool mute);
   void ToggleMute(void);
 
+  ReplayGainSettings& GetReplayGainSettings() { return m_replayGainSettings; }
```

Comment:
We should probably make this const right?

Reply:
maybe. i'll have to investigate if it can be const when i'm at my kodi dev box.

Reply:
yup. constified.